crates/sargon/src/profile/v100/factors/factor_source_category.rs

```diff
@@ -0,0 +1,17 @@
+use crate::prelude::*;
+
+/// An enum representing the **category** of a `FactorSource`/`FactorSourceKind`.
+#[derive(Clone, Copy, Debug, PartialEq, Eq)]
+pub enum FactorSourceCategory {
```

Comment:
FactorSourceKindCategory imo

Reply:
I thought about this but went with `FactorSourceCategory` because `kind` and `category` are synonyms, so it sounded a bit weird. Also, given a `FactorSource` can be categorized as well (in fact, it is how we use it) I think this makes more sense.